pkg/volume/portworx/portworx_util.go

```diff
@@ -23,16 +23,18 @@ import (
 	volumeclient "github.com/libopenstorage/openstorage/api/client/volume"
 	osdspec "github.com/libopenstorage/openstorage/api/spec"
 	osdvolume "github.com/libopenstorage/openstorage/volume"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/kubernetes/pkg/api/v1"
 	"k8s.io/kubernetes/pkg/volume"
 )
 
 const (
-	osdMgmtPort      = "9001"
-	osdDriverVersion = "v1"
-	pxdDriverName    = "pxd"
-	pwxSockName      = "pwx"
-	pvcClaimLabel    = "pvc"
+	osdMgmtPort         = "9001"
+	osdDriverVersion    = "v1"
+	pxdDriverName       = "pxd"
+	pwxSockName         = "pwx"
+	pvcClaimLabel       = "pvc"
+	labelNodeRoleMaster = "node-role.kubernetes.io/master"
```

Comment:
This is Portworx specific label and thus should have `portworx` prefix. In addition, masters and nodes can be separate machines, `/master` is not a good name here... Perhaps `portworx.kubernetes.io/driver`? Feel free to suggest a better name.